src/app/flashcard/components/pages/create-flashcard-page/create-flashcard-page.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Observable, of, Subscriber, Subscription } from 'rxjs';
+import { Topic } from 'src/app/flashcard/model/topic';
+import { TopicService } from 'src/app/flashcard/service/topic.service';
+import { FlashcardService } from '../../../service/flashcard.service'
+
+@Component({
+  selector: 'app-create-flashcard-page',
+  templateUrl: './create-flashcard-page.component.html',
+  styleUrls: ['./create-flashcard-page.component.css']
+})
+export class CreateFlashcardPageComponent implements OnInit {
+
+  topics: Topic[] = [];
+  subscription: Subscription;
+  anonymous: boolean = false;
+
+  form: any = {
+    creator: null,
+    topic: {},
+    question: null,
+    questionDifficultyTotal: null
+  }
+
+  constructor(private flashcardService: FlashcardService, private topicService: TopicService) {
+    this.subscription = new Observable<any>().subscribe();
+    this.topicService.getAll().subscribe(res => this.topics = res);
+  }
+
+
+  ngOnInit(): void {
+  }
+
+  onSubmit(): void {
+
+    if (this.anonymous) {
```

Comment:
For now, all flashcard questions are anonymous by default. We should still keep track of who the creator is for admin viewing purposes, but their name will never show up on the flashcard display